client/src/includes/tabs.js

```diff
@@ -230,6 +230,13 @@ class Tabs {
         }
       }
     });
+
+    // Dispatch tab-changed on window load
+    window.onload = () => {
+      if (this.tabContainer) {
```

Comment:
I don’t see how this could ever not be set? We’re already relying on `this.tabContainer` being defined elsewhere so I don’t see what this protects us from.

Reply:
Ah oops you're right, i've removed this check now!